test/makeup/erlang_lexer/erlang_lexer_tokenizer_test.exs

```diff
@@ -101,10 +101,112 @@ defmodule ErlangLexerTokenizer do
       assert lex(~s/<<"string">>/) == [
                {:punctuation, %{}, "<<"},
                {:punctuation, %{}, "\""},
-               {:name_symbol, %{}, "string"},
+               {:string_symbol, %{}, "string"},
                {:punctuation, %{}, "\""},
                {:punctuation, %{}, ">>"}
              ]
     end
   end
+
+  describe "atoms" do
+    test "are tokenized as such" do
+      assert lex("atom") == [{:string_symbol, %{}, "atom"}]
+    end
+
+    test "are tokenized as such even when quoted" do
+      assert lex("'atom'") == [{:string_symbol, %{}, "'atom'"}]
+      assert lex("'atom atom'") == [{:string_symbol, %{}, "'atom atom'"}]
+      assert lex("'atom+atom'") == [{:string_symbol, %{}, "'atom+atom'"}]
+      assert lex("'atom@atom'") == [{:string_symbol, %{}, "'atom@atom'"}]
+      assert lex("'atom123atom'") == [{:string_symbol, %{}, "'atom123atom'"}]
+    end
+
+    test "does not tokenize invalid characters as atom (\\n, ', \\)" do
+      assert Enum.member?(lex("atom\n"), {:string_symbol, %{}, "atom"})
+      assert Enum.member?(lex("atom'"), {:string_symbol, %{}, "atom"})
+      assert Enum.member?(lex("atom\\"), {:string_symbol, %{}, "atom"})
```

Comment:
You can use `in` here and in the assertions below, it will make them a bit more readable:

```suggestion
      assert {:string_symbol, %{}, "atom"} in lex("atom\\")
```

Reply:
[done](https://github.com/tmbb/makeup_erlang/compare/1b0d9c865c495008261a4f2f7c9a46a243a489f8..78ad4e00adba1793f54091c4d15c60b47dcd4fad)